specs/proposal/542678_improve_bundle_image_provided_shape/improve_bundle_image_provided_shape.textile

```diff
@@ -41,7 +41,7 @@ This feature uses the existing API and does not need new API or API changes.
 
 h3. User Interface Changes
 
-The extension point tags _colorAttribute_, _borderColorAttribute_ and __borderSizeAttribute__ will now provide a sub attribute (not mandatory).
+The extension point tags @colorAttribute@, @borderColorAttribute@ and @borderSizeAttribute@ will now provide a sub attribute (not mandatory).
 
 h3. Documentation Changes
```

Comment:
>  * In developer documentation, the documentation of the "provided bundled image shape extension point":https://www.eclipse.org/sirius/doc/developer/extensions-provide_custom_bundled_image_shape.html needs to be updated.

This is wrong now. It must be marked as deprecated in favor of the new extension point that must be documented.

Reply:
Done